From 57ec2296445225a11706b1a1eaae56b37a06c9eb Mon Sep 17 00:00:00 2001
From: Alexander Aring <alex.aring@gmail.com>
Date: Fri, 28 Mar 2014 12:04:23 +0100
Subject: [PATCH] libusb: needs librt and libpthread

Original commit msg from older libusb package:

Add missing dependencies.

20100603 rsc: The patch was closed upstream, and I don't
         remember a use case any more :-( Let's keep it
         here for the moment, until we have a user.
20100101 rsc: added as http://libusb.org/ticket/24

Signed-off-by: Alexander Aring <alex.aring@gmail.com>
---
 libusb-1.0.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libusb-1.0.pc.in b/libusb-1.0.pc.in
index aa023b3..eded053 100644
--- a/libusb-1.0.pc.in
+++ b/libusb-1.0.pc.in
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: libusb-1.0
 Description: C API for USB device access from Linux, Mac OS X, Windows and OpenBSD/NetBSD userspace
 Version: @VERSION@
-Libs: -L${libdir} -lusb-1.0
+Libs: -L${libdir} -lusb-1.0 -lrt -lpthread
 Libs.private: @LIBS@
 Cflags: -I${includedir}/libusb-1.0
-- 
1.9.2

